docs/blockchain/contracts/nft-contract/nft-actions/acptfctofr.a.experimental.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'acptfctofr.a'
+order: 41
+
+---
+
+# acptfctofr.a
+
+Accept the buy offer made on the token factory.
```

Comment:
I think that in all action pages, using `token` term is a bit confusing, "NFT" is better because "NFT"  and "NFT factory" are used in the existing other pages.